elmo/elmo-rss.el

```diff
@@ -164,8 +164,22 @@ Setting this to true will annoy the pedants."
    (member (xml-get-attribute-or-nil node 'rel)
            '(nil "alternate" "enclosure"))))
 
+(defun elmo-rss-parse-atom-author (author)
+  (let ((author-name
+         (and author
+              (car (xml-node-children
+                    (assoc 'name (xml-node-children author))))))
+        (author-email
+         (and author
+              (car (xml-node-children
+                    (assoc 'email (xml-node-children author)))))))
+    (cond
+      (author-name
+       (concat (elmo-rss-utf-8 author-name) " <" (or author-email "") ">"))
```

Comment:
It would be better not adding `"<>"` when `author-email` is `nil`.

Reply:
Done.